microshift_storage/microshift-storage-migration.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="microshift-storage-migration"]
+= Storage migration using the Kube Storage Version Migrator
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-storage-plugin-overview
+
+toc::[]
+
+Storage version migration is used to update existing in cluster objects to the latest version. The Kube Storage Version Migrator embedded controller is used in {product-title} to migrate resources with out having to recreate those resources. Either you or a controller can create a `StorageVersionMigration` custom resource (CR) that will request a migration through the Migrator Controller.
```

Comment:
"Storage version migration is used to update existing in cluster objects to the latest version."
Is it updating the existing version?

Reply:
Yeah, it updates the objects from whatever they are to the latest version.